src/main/java/com/redhat/cloud/notifications/db/NotificationResources.java

```diff
@@ -12,7 +12,6 @@
 import io.vertx.core.json.JsonObject;
 import reactor.core.publisher.Flux;
 import reactor.core.publisher.Mono;
-
```

Comment:
These shouldn't cause errors - this is the default type from IntelliJ.

Reply:
With the separator lines:

```
[INFO] --- maven-checkstyle-plugin:3.1.1:check (validate) @ notifications-backend ---
[INFO] Beginne Prüfung...
[ERROR] /Users/hrupp/insights/notifications-backend/src/main/java/com/redhat/cloud/notifications/webhooks/WebhookProcessor.java:17: Zusätzliche Trennung in der Importgruppe vor 'javax.enterprise.context.ApplicationScoped' [ImportOrder]
Prüfung beendet.

```

Reply:
Ah, it's because of your German language:

```xml
        <module name="SuppressionXpathSingleFilter">
            <property name="checks" value="ImportOrder"/>
            <property name="message" value="Extra separation in import group before 'java?..*"/>
        </module>
```

So that one isn't matching.

Reply:
What do you mean and suggest?

Reply:
I mean that checkstyle check modification doesn't understand non-English locales for error messages.

Reply:
And that's why it doesn't work on your machine.